ci-operator/config/redhat-developer/rhdh/redhat-developer-rhdh-release-1.7.yaml

```diff
@@ -29,7 +29,7 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.18"
-  skip_if_only_changed: ^docs/|showcase-docs/|^\.changeset/|CONTRIBUTING\.md|OWNERS|README\.md|USAGE_DATA\.md|^scripts/|^\.github/|\.threatmodel/|^ui/|\.md|\\.mdc|\\.toml$
+  skip_if_only_changed: ^docs/|rpms\.lock\.yaml|showcase-docs/|^\.changeset/|CONTRIBUTING\.md|OWNERS|README\.md|USAGE_DATA\.md|^scripts/|^\.github/|\.threatmodel/|^ui/|\.md|\\.mdc|\\.toml$
```

Comment:
looks good but can you update the 1.8 files too?

eg., https://github.com/openshift/release/blob/3760c79388d31368a56a2e15e8139709792a4273/ci-operator/config/redhat-developer/rhdh/redhat-developer-rhdh-release-1.8.yaml